cns/service/main.go

```diff
@@ -316,6 +377,22 @@ func main() {
 		go httpRestService.SendNCSnapShotPeriodically(cnsconfig.TelemetrySettings.SnapshotIntervalInMins, stopSnapshots)
 	}
 
+	// If CNS is running on managed DNC mode
+	if privateEndpoint != "" && infravnet != "" && nodeID != "" {
+		httpRestService.SetOption(acn.OptPrivateEndpoint, privateEndpoint)
+		httpRestService.SetOption(acn.OptInfrastructureNetwork, infravnet)
+		httpRestService.SetOption(acn.OptNodeID, nodeID)
+
+		registerNode(httpRestService, privateEndpoint, infravnet, nodeID)
+		go func(ep, vnet, node string) {
+			// Periodically poll (30s) DNC for node updates
+			for {
+				<-time.NewTicker(time.Second * 30).C
```

Comment:
move 30 sec to a const

Reply:
it's only used here

Reply:
my comment in general is to not hardcode values like 5 seconds, 30 seconds. Instead declare those as variable like syncNodeInterval.

Reply:
I just finished reviewing the DNC PR. We are retrieving all the NCs every 30 seconds. let's expose this as a configurable param via config file. If we see we are hitting cosmosdb RU limit due to NC processing in DNC we can increase this w/o needing to update the code.